plugins/copr.py

```diff
@@ -491,18 +497,27 @@ def _guess_chroot(self):
         elif "Amazon Linux" in dist[0]:
             chroot = "amazonlinux-{}-{}".format(dist[1], distarch if distarch else "x86_64")
         else:
-            chroot = ("epel-{}-{}".format(dist[1].split(".", 1)[0], distarch if distarch else "x86_64"))
+            releasever = dist[1].split(".", 1)[0]
+            arch = distarch if distarch else "x86_64"
+
+            if available_chroots:
+                guesses = ['epel', 'rhel']
+                if "CentOS Stream" in dist:
+                    guesses.insert(1, 'centos-stream')
```

Comment:
https://github.com/rpm-software-management/dnf-plugins-core/pull/525#issuecomment-1985507234 showed me there's also the `centos-stream+epel-next` chroot. Should we support that too?
```suggestion
                if "CentOS Stream" in dist:
                    guesses = ['centos-stream+epel-next', 'epel', 'centos-stream', 'rhel']
                else:
                    guesses = ['epel', 'rhel']
```

Reply:
The `centos-stream+epel-next` chroots are tricky.  For versions 8 and 9 (the only ones we have EPEL Next), real systems should be one of:

* RHEL + EPEL
* CentOS Stream + EPEL + EPEL Next

But in mock-core-configs we have all of these:

* `rhel+epel`
* `centos-stream+epel`
* `centos-stream+epel-next`

For versions 8 and 9, EPEL on CentOS Stream means both EPEL and EPEL Next.  Using only EPEL without EPEL Next on CentOS Stream is not valid and will result in uninstallable packages (albeit rarely).  I think the `centos-stream+epel-next` chroots should be retired and `centos-stream+epel` chroots should be updated to also include the `epel-next` template (matching what `centos-stream+epel-next` is now).  If we do that, there won't be any need to include logic in the copr plugin for `centos-stream+epel-next`.

Also, EPEL 8 Next is already EOL, EPEL 9 Next goes EOL in about two years, and EPEL 10 uses a totally different structure that obsoletes EPEL Next.